packages/docs/components/page-configs/va-avatar/api-options.ts

```diff
@@ -0,0 +1,26 @@
+import { ManualApiOptions } from 'vuestic-ui/src/services/api-docs/ManualApiOptions'
```

Comment:
We don't need empty avatar in examples at all as it serves no purpose in app.
![image](https://user-images.githubusercontent.com/5394573/82836592-75ccdb00-9ecf-11ea-8772-7357f8afa7a3.png)
So I recommend to either remove or replace examples with content-full avatars.